### PR TITLE
Improve README with common errors encountered

### PR DESCRIPTION
**Pull request by galderz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,3 +80,96 @@ Finally, all of the OpenWhisk resources can be shutdown using the
 template:
 
     oc process -f template.yml | oc delete -f -
+
+## Common Problems
+
+### Runtime error when setting `apihost` for `wsk`
```
Comment (jcrossley3) [inline README.md]: Couldn't we just add quotes to the command elsewhere in the README and eliminate this section? :)

File: README.md [hunk 2]
```diff
@@ -80,3 +80,96 @@ Finally, all of the OpenWhisk resources can be shutdown using the
...
+    Client key
+    whisk auth                  789c46b1-...
+    whisk API host              http://openwhisk-openwhisk.192.168.64.8.nip.io
+    whisk API version           v1
+    whisk namespace             _
+    whisk CLI version           2018-02-28T21:13:48.864+0000
+    whisk API build             2018-01-01T00:00:00Z
+    whisk API build number      latest
+
+API host should only contain the host name, no `http://` in front.
```
Comment (jcrossley3) [inline README.md:188]: How did the `http://` prefix get there in the first place? I'd rather solve that, if possible.

**Review by jcrossley3 — CHANGES_REQUESTED** [review]
Thanks for the feedback, Galder. Just a coupla questions...

File: README.md (hunk 2 above)
Comment (galderz) [reply README.md:188]: No idea

File: README.md (hunk 1 above)
Comment (galderz) [reply README.md]: Dunno, does it work fine with bash and other shells?

Comment (jcrossley3) [reply README.md]: It doesn't bother bash. And if it appeases zsh, too, that's 2 down! :)

Comment (galderz) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (galderz) [reply README.md:188]: This must has been my own fault.

**Comment by galderz** [comment]
This is ready now

**Pull request merged**